src/Console/Command/FixCommand.php

```diff
@@ -347,6 +347,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
             }
         }
 
+        if ($config->usingCache()) {
```

Comment:
if we put `null !== $stdErr` on start we could save 2 methods calls and one IO call